Scripts/uetorch.lua

```diff
@@ -508,6 +510,7 @@ uetorch.SetActorVisible = utlib.SetActorVisible
 uetorch.SetActorVelocity = utlib.SetActorVelocity
 uetorch.SetActorAngularVelocity = utlib.SetActorAngularVelocity
 uetorch.SetActorScale3D = utlib.SetActorScale3D
+uetorch.DestroyActor = utlib.DestroyActor
```

Comment:
I read the code but I can't make sense of what you're doing. What is this magic constant 0.236? It looks like on iteration 0 you're deleting a bunch of stuff, then taking screenshots to determine if hidden.

I think you can determine if an object X is hidden like so

```
X = uetorch.GetActor('X')
seg = uetorch.ObjectSegmentation({X})
hidden = torch.max(seg)
```